sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -21,6 +24,10 @@
 import com.azure.cosmos.models.CosmosClientTelemetryConfig;
 import com.azure.cosmos.models.CosmosPermissionProperties;
 
+// TODO: Use slf4j Logger and LoggerFactory instead of the core ClientLogger?
+// import org.slf4j.Logger;
+// import org.slf4j.LoggerFactory;
+
```

Comment:
ClientLogger (a core API class) is being used in the repo ... but so is log4j factory/logger.
I am not sure what would be the better option. Seems one should be using ClientLogger.

Reply:
Switched to log factory like used in the rest of cosmos SDK